packages/notifier/src/notifier.js

```diff
@@ -30,24 +30,25 @@ export const makeNotifier = sharableInternalsP => {
   });
 };
 
+const deferInitialState = Symbol('defer initial NotifierKit state');
```

Comment:
Although it doesn't work in this case, I generally like this technique of introducing a unique value and using it in an encapsulated manner to distinguish it from any value that users of this abstraction could have passed in. However, it is important that any value used in this way cannot leak outside this module. Otherwise, a caller that happens to handle such a value (whether maliciously or accidentally) can confuse the abstraction by passing it back in. This encapsulation needs to be robustly preserved under maintenance. 
   * The value declaration itself should be commented to make it clear that the value must not leak. When I use this technique I try to always remember to add this comment.
   * The code that actually uses the dangerous value should be written to *obviously* not leak it. There should be little code between where it is introduced and where it is tested and erased. It should be trivial for a reviewer to verify that it does not leak.
   * CI reveals that it actually did leak, which genuinely surprised me! Because of the amount of code between its introduction and its testing/erasure, I have not yet even spotted how it leaked.